### ISSUE TITLE
Build Exception for Flutter 3.22.3-stable

### ISSUE THREAD

**Issue by LukasKompatscher** [davigmacode/flutter_flexi_chip#1]
```bash
../../../../.pub-cache/hosted/pub.dev/flexi_chip-1.0.8/lib/src/event.dart:67:7: Error: The superclass, 'WidgetEventController', has no unnamed constructor that takes no arguments.
class FlexiChipEventController extends WidgetEventController {}
      ^
../../../../.pub-cache/hosted/pub.dev/flexi_chip-1.0.8/lib/src/widget.dart:727:25: Error: Too many positional arguments: 1 allowed, but 2 found.
Try removing the extra positional arguments.
      updateWidgetEvents(oldWidget.eventsController, widget.eventsController);
```
      
  Flutter doctor:
  ```bash
  ❯ flutter doctor -v                                            
[✓] Flutter (Channel stable, 3.22.3, on macOS 14.6 23G80 darwin-arm64, locale en-IT)
    • Flutter version 3.22.3 on channel stable at /Users/lukas/.local/share/mise/installs/flutter/3.22.3-stable
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision b0850beeb2 (2 weeks ago), 2024-07-16 21:43:41 -0700
    • Engine revision 235db911ba
    • Dart version 3.4.4
    • DevTools version 2.34.3

[✓] Android toolchain - develop for Android devices (Android SDK version 35.0.0)
    • Android SDK at /Users/lukas/Library/Android/sdk
    • Platform android-35, build-tools 35.0.0
    • Java binary at: /Users/lukas/Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 17.0.11+0-17.0.11b1207.24-11852314)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 15.4)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Build 15F31d
    • CocoaPods version 1.15.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2024.1)
    • Android Studio at /Users/lukas/Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 17.0.11+0-17.0.11b1207.24-11852314)

[✓] IntelliJ IDEA Ultimate Edition (version 2024.1.4)
    • IntelliJ at /Users/lukas/Applications/IntelliJ IDEA Ultimate.app
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart

[✓] VS Code (version 1.88.1)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.92.0

[✓] Connected device (4 available)
    • sdk gphone64 arm64 (mobile)     • emulator-5554         • android-arm64  • Android 14 (API 34) (emulator)
    • macOS (desktop)                 • macos                 • darwin-arm64   • macOS 14.6 23G80 darwin-arm64
    • Mac Designed for iPad (desktop) • mac-designed-for-ipad • darwin         • macOS 14.6 23G80 darwin-arm64
    • Chrome (web)                    • chrome                • web-javascript • Google Chrome 127.0.6533.74

[✓] Network resources
    • All expected network resources are available.

  ```

**Comment by Giovanni-Fantin** [comment]
I've been experiencing the same issue since this morning

**Comment by LukasKompatscher** [comment]
@Giovanni-Fantin In the meantime you can use [ChoiceChip](https://api.flutter.dev/flutter/material/ChoiceChip-class.html)

**Comment by Giovanni-Fantin** [comment]
I think the problem is in this package:  widget_event 1.10.1 (was 1.9.0), that had an upgrade this morning, and another  widget_event 1.10.2 (was 1.10.1) a few minutes later. And flexi_chip depends on widget_event

**Comment by davigmacode** [comment]
Hi, sorry for the inconvenience. There was an issue with `widget_event` v1.10.* which should have been a breaking change to v2. I've reverted the code to v1.9.0 and released it as v1.11.0. Please run `flutter packages upgrade` to resolve the error.

If you're using the `chips_choice` package, consider switching to `choice` package for significant improvements.
